test/salesforce/event-flow/main.go

```diff
@@ -248,7 +184,7 @@ func TestCredentialsLegacy(conn *salesforce.Connector, ctx context.Context, uniq
 			Label:                       "TestNamedCredentialLegacy" + uniqueString,
 
 			// below are legacy fields
-			Endpoint:      "arn:aws:us-east-2:381491976069",
+			Endpoint:      "arn:aws:US-EAST-2:381491976069",
```

Comment:
Does case sensitivity matter? If so, would be great to add a comment here about it

Reply:
same question. Almost universally, AWS uses lower case for region identifiers like this, not sure where this case change is coming from.

Reply:
@RajatPawar @eberle1080 
Salesforce complains for lower case hahah

```
-----------------Testing Named Credential Legacy-----------------
time=2024-10-21T15:09:20.932-07:00 level=ERROR msg="Error named cred" error="bad request: We can't create the named credential because the AWS region that the named credential references is invalid. Make sure the AWS region is in upper case and is supported."
```

Reply:
https://help.salesforce.com/s/articleView?id=release-notes.rn_messaging_event_relays_aws_region.htm&language=en_US&release=252&type=5
Just SF things.